db/re/item_db_etc.yml

```diff
@@ -21557,6 +21583,36 @@ Body:
     Type: Etc
     Buy: 10
     Weight: 100
+  - Id: 6720
+    AegisName: Status_Reset_Coupon
+    Name: Status Reset Coupon
+    Type: Cash
```

Comment:
This item is currently type Cash, why putting it in `item_db_etc.yml` ?

Reply:
Sry it is etc, not cash. I forgot to change it